### PR TITLE
i18n(ja): Add Japanese `install/manual.md`

### PR DESCRIPTION
**Pull request by kyosuke**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

- Translated the manual install page into Japanese.

### CODE REVIEW

File: src/pages/ja/install/manual.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+プロジェクトのルートに `astro.config.mjs` を作成し、その中に以下のコードをコピーします。
+
+```
+import { defineConfig } from 'astro/config';
+
+// https://astro.build/config
+export default defineConfig({});
+```
+
+ReactやSvelteなどの[UIフレームワークコンポーネント](/ja/core-concepts/framework-components/)や、TailwindやPartytownなどのツールをプロジェクトに組み込む場合は、ここで[手動でインテグレーションを取り込んで設定](/ja/gides/integrations-guide/)します。
```
Comment (sarah11918) [inline src/pages/ja/install/manual.md:115]:
```suggestion
ReactやSvelteなどの[UIフレームワークコンポーネント](/ja/core-concepts/framework-components/)や、TailwindやPartytownなどのツールをプロジェクトに組み込む場合は、ここで[手動でインテグレーションを取り込んで設定](/ja/guides/integrations-guide/)します。
```

Comment (sarah11918) [reply src/pages/ja/install/manual.md:115]: Thank you, @hippotastic !  😉

**Review by yuto343 — APPROVED** [review]
LGTM！

**Comment by sarah11918** [comment]
Thank you for your review, @yuto343 ! Nice to have you on Team Docs! 🥳

**Pull request merged**